x/feegrant/client/cli/tx.go

```diff
@@ -138,8 +138,8 @@ Examples:
 
 					periodic := types.PeriodicFeeAllowance{
 						Basic:            basic,
-						Period:           types.ClockDuration(time.Duration(periodClock) * time.Second),
-						PeriodReset:      types.ExpiresAtTime(periodReset),
+						Period:           time.Duration(periodClock) * time.Second,
+						PeriodReset:      time.Now().Add(time.Duration(periodClock) * time.Second),
```

Comment:
maybe extract some utility function since it's being used in client/cli and client/testutil